### ISSUE TITLE
EntityTracker3D not working with "PLAYER" trackingType

### ISSUE THREAD

**Issue by Zephyranthes28** [StarChasers/OCGlasses#52]
I have been trying to get an object to render on a player, in this case, a white sphere, and using the 'Living' or 'Player' types, it doesn't work. However, it does work with 'All'. 

I couldn't figure out what the issue was myself, however I'm not very well versed in lua so it could be user error. 

Player standing next to a chick-
https://imgur.com/Om42lFV

Here's the snippet I ran to generate that-
```
component = require("component")

fh = io.open("/home/test.obj", "r")
objData = fh:read("*a")
fh:close()

g = component.glasses.addEntityTracker3D()

g.addScale(0.5, 0.5, 0.5)
g.loadOBJ(test)
g.setTrackingType("LIVING", 32)
```

**Comment by ben-mkiv** [comment]
thanks for your bugreport

**Comment by ben-mkiv** [comment]
bug is fixed in 2.0.1 release which is now on curseforge
